### DISCUSSION TITLE
[Console MongoDB CRUD Section]: collection tags

### DISCUSSION THREAD

**Discussion by marco-basso** [mia-platform/community#256] [Q&A] [answered]
### Description

Hi! While refining the documentation of a project I stumble on the following problem. I would like to tag the collections listed within the MongoDB CRUD Console section with more than one tag.

Inside the `api-console-config.json` file there is the `tags` array field associated to each collection configuration:

```
"collections": {
  ...
  "collection-name": {
    "id": "id",
    "description": "description",
    "name": "collection-name",
    "tags": [ .. ],
    "fields": [ .. ],
    "internalEndpoints": [ .. ],
    "type": "collection",
    "indexes": [ .. ],
  },
  ...
}
```
From the console it is possible to define only one of them. I tried to configure more than one tag on the configuration and it didn't broke the console loading, but only the first tag was shown.

Will it be possible to visualize and insert multiple tags from the console?

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, when we introduced tags we decided to allow for multiple tags in the data model however, for the sake of UX simplicity, we also decided to only support one for the moment; what would you like multiple tags for? Let's think about the need so that we can understand how to show them in the Console and allow for having multiple tags support.

@albertotessarotto FYI

**Reply by marco-basso** [reply]
We are working on a tool that builds a markdown documentation from the console configurations file.

The client that requested such documentation asked for the data source of our collections. 

We decided to insert these sources as tags to avoid manual operations on the markdown documentation build.

Some tables happen to have data from different sources (requiring more than one tag) and therefore we found this need.

**Reply by fredmaggiowski** [reply]
By "console configurations file" do you mean the `api-console-config.json` file in the project configuration repository?
